website/docs/guides/1.1.0-upgrade-guide.html.markdown

```diff
@@ -8,73 +8,87 @@ MongoDB Atlas Provider 1.1.0: Upgrade and Information Guide
+* [`mongodbatlas_privatelink_endpoint`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/privatelink_endpoint) and [`mongodbatlas_privatelink_endpoint_service`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/privatelink_endpoint_service) now support Google Cloud'd Private Service Connect
+* Projects created with [`mongodbatlas_project`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/project)` can now have default alert creation disabled by setting `with_default_alerts_settings` to false.
+* [`mongodbatlas_search_index`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/search_index) now supports synonyms.
+* Atlas Serverless deployments, now in preview, can now be created with [`mongodbatlas_serverless_instance`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/serverless_instance)
 
-There are two possible options to upgrade from v1.0.2 to v1.1.0 from the existing resources `mongodbatlas_cloud_provider_snapshot_backup_policy` to the new ones
+Changes:
+In addition, there are changes that will require modifications to your Terraform configuration between earlier versions of the provider and this one.
 
+* `mongodbatlas_cloud_backup_snapshot`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/cloud_backup_snapshot) replaces the now deprecated `mongodbatlas_cloud_provider_snapshot`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/cloud_provider_snapshot).
+* `mongodbatlas_cloud_backup_snapshot_restore_job`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/cloud_backup_snapshot_restore_job) replaces the now deprecated `mongodbatlas_cloud_provider_snapshot_restore_job`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/cloud_provider_snapshot_restore_job).
 
-### Option 1:  Remove the existing and import the new resource
+1.1.0 also includes general improvements and bug fixes. See the [CHANGELOG](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/CHANGELOG.md) for more specific information.
 
-1. Remove your existing `mongodbatlas_cloud_provider_snapshot` resources from terraform state
-   **NOTE** before doing any terraform state change please do a backup
+## Migration from Cloud Provider Snapshot to Cloud Backup Snapshot
 
-```bash
-$ terraform state rm mongodbatlas_cloud_provider_snapshot.test
-```
+The term Cloud Provider has been deprecated in favor of the term Cloud Backup.  In order to support this change within the Terraform provider, related resources and data sources with the older terminology have been deprecated and new resources and data sources added to support the new terminology.  In order to upgrade from the previous resources to the new you'll need to select from one of the following two options:
 
```

Comment:
the difference between option 1 and option 2 is not clear to me. it looks like maybe if users have used the old providers, they must complete option 2. if they don't they can use option 1. is this accurate?

Reply:
One requires just changing the names in the state file. Not a great practice in the Terraform world but works. The second is the more official way, but slightly harder.   It requires fully removing the old resources from state and then re-importing them.